inc/log.class.php

```diff
@@ -294,6 +304,16 @@ static function showForItem(CommonDBTM $item, $withtemplate = 0) {
       $header .= "<th>"._n('Field', 'Fields', 1)."</th>";
       //TRANS: a noun, modification, change
       $header .= "<th>"._x('name', 'Update')."</th>";
+      $header .= "<th class='log-toolbar'>";
+      if (isset($_GET['filters'])) {
+         $header .= "<i class='fas fa-filter log-toolbar-item show_log_filters active'></i>";
```

Comment:
add title attribute to both i.fas tag to permits an hover indication (and accessibility)